### PR TITLE
config/print/index.md: add Brother laser pkg and lpadmin group requir…

### PR DESCRIPTION
**Pull request by basploeger**
…ement

Before submitting a pull request, please read [CONTRIBUTING](https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md); pull requests that do not meet the criteria described there will not be merged. Note that this repository's CONTRIBUTING contains information specific to this repository, and is not the same as CONTRIBUTING for the void-packages repository.

We prioritise pull requests involving information specific to Void over those involving information applicable to Linux in general.

### CODE REVIEW

File: src/config/print/index.md [hunk 1]
```diff
@@ -66,7 +67,8 @@ install the `avahi` package and enable the `avahi-daemon` service.
 
 To configure the printer using the CUPS web interface, navigate to
 <http://localhost:631> in a browser. Under the "Administration" tab, select
-"Printers > Add Printer".
+"Printers > Add Printer". Log in using an account that is in the `lpadmin`
```
Comment (bobertlo) [inline src/config/print/index.md:70]: This might read more clearly if it was at the end of the sentence about logging into CUPS? Not a big deal really.

Comment (basploeger) [reply src/config/print/index.md:70]: Well, if I follow these instructions, I get the login dialog after clicking the 'Add Printer' button. I rephrased it a bit to improve readability.

**Comment by basploeger** [comment]
Please take another look. Thanks!

**Comment by basploeger** [comment]
@bobertlo PTAL

**Comment by bobertlo** [comment]
LGTM I can't merge it though just a friendly review :)

**Pull request merged**